paparazzi-preview-test-junit/src/main/java/app/cash/paparazzi/preview/Utils.kt

```diff
@@ -0,0 +1,16 @@
+// Copyright Square, Inc.
+package app.cash.paparazzi.preview
+
+import androidx.compose.runtime.Composable
+import app.cash.paparazzi.Paparazzi
+import app.cash.paparazzi.annotations.PaparazziPreviewData
+
+internal fun Paparazzi.snapshotDefault(
+  previewData: PaparazziPreviewData.Default,
+  name: String?,
+  wrapper: @Composable (@Composable () -> Unit) -> Unit = { it() }
+) {
+  snapshot(name) {
+    wrapper { previewData.composable() }
```

Comment:
What does the wrapper actually do? Why not just call `previewData.composable()` inside `snapshots`'s lambda?

Reply:
Do you have examples for this? I'm having trouble thinking of ways I would use this API in practice

Reply:
let me take a look